.gitmodules

```diff
@@ -29,3 +29,6 @@
 [submodule "resource/SingleFileZ"]
 	path = resource/SingleFileZ
 	url = https://github.com/gildas-lormeau/SingleFileZ.git
```

Comment:
We can remove this now, right?

Reply:
Yes. I didn't know if there was a good or bad way of doing that so left it.